lib/src/st_rx_video_session.c

```diff
@@ -2641,6 +2643,8 @@ static int rv_attach(struct st_main_impl* impl, struct st_rx_video_sessions_mgr*
     info("%s(%d), hdr_split enabled in ops\n", __func__, idx);
   }
 
+  s->st20_linesize = ops->width * s->st20_pg.size / s->st20_pg.coverage;
+  if (ops->linesize > s->st20_linesize) s->st20_linesize = ops->linesize;
```

Comment:
Pls return error if ops->linesize is invalid(width * s->st20_pg.size / s->st20_pg.coverage)